.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
I am not sure if we can do this, but if you put something like `bug` in the labels of the template would issues created from this template get that label by default? If so, that seems useful (same for the feature request one)